docs/install/README.md

```diff
@@ -7,30 +7,54 @@ description: There are several different ways you can install and interact with
 
 IPFS is a collection of protocols, packages, and specifications that allow computers to send and receive data. Because of this, users can interact with and use IPFS in many different ways. A developer building network applications will use a different set of tools to interact with IPFS than someone who wants to store files on IPFS. Pick the one that best suits what you're here to do.
 
-## Publish files with a pinning service
+Looking for an easy and opinionated way to get started with IPFS [Mainnet](../concepts/glossary.md#mainnet)? Try any of the options listed below:
 
-Do you want to quickly and easily publish content with IPFS without complex tools? See the [Publish with IPFS quickstart](../quickstart/publish.md), where you'll learn how to use pinning services to pin files to the IPFS network.
+## User-Friendly Options
```

Comment:
I think this should be called "No-Code Tools" or "Desktop Apps" — all options should be user-friendly.

Reply:
a91b6d697084c1c3c859722258a9c5523abe3299 renamed to "Desktop Users"